dev_playbook.yml

```diff
@@ -38,6 +38,7 @@
         path: "{{ item }}"
       with_items:
         - "{{ galaxy_config_dir }}/mail"
+        - "{{ galaxy_config_dir }}/tool_panel_filters"
```

Comment:
could you please add this to `galaxy_extra_privsep_dirs` in galaxyservers.yml and remove this task (lines 35-40) instead?